### DISCUSSION TITLE
How to go about testing a Zephyr HXM BT heart rate monitor?

### DISCUSSION THREAD

**Discussion by rspilk** [OpenTracksApp/OpenTracks#1577] [Q&A] [answered]
The legacy project by Plnk42 supported this sensor and I was wondering if anyone knew why it was removed or how i might go about trying to introduce it again

Thanks,

 https://github.com/Plonk42/mytracks/blob/master/myTracks/src/main/java/com/google/android/apps/mytracks/services/sensors/ZephyrMessageParser.java

**Comment by dennisguse** [comment, marked as answer]
I removed it, because this type of sensor was even old 4 years ago.
And it was easier to remove Bluetooth Classic and Ant while replacing it with Bluetooth Low Energy.

IIRC one problem back then was that there was no standard interface for these sensors on Bluetooth Classic.
Every company created their own and that is kind of annoying to maintain.
And for Bluetooth Low Energy that changed as there are standards to use.

TL;DR it can be re-implemented if somebody has the time.
I don't see too much benefit in doing so as all modern sensors are Bluetooth Low Energy.
